docs/entities/contacts/NextFriendForIncompetentPersonContact(DOMESTIC).md

```diff
@@ -46,6 +46,18 @@
           name: "max"
           args: 
             limit: 500
+    contactId: 
```

Comment:
I see this repeated for a bunch of different contacts in here, and I guess I'm just looking to understand better:
* how these contacts are different from others
* could they be consolidated with a lookup table of sorts for the various types or for International / Domestic?

Reply:
The code has a contact factory that instantiates the correct type. These markdown files are only used to communicate requirements with the legacy migration team.